src/main/java/space/pxls/server/WebHandler.java

```diff
@@ -1714,7 +1716,8 @@ public void info(HttpServerExchange exchange) {
             Math.min(App.getConfig().getInt("chat.characterLimit"), 2048),
             App.getConfig().getBoolean("chat.canvasBanRespected"),
             App.getConfig().getStringList("chat.bannerText"),
-            App.getConfig().getBoolean("oauth.snipMode")
+            App.getConfig().getBoolean("oauth.snipMode"),
+            emoji
```

Comment:
I think it's better to read the `chat.customEmoji` list on each request like the rest of these parameters are doing. That way the emotes are updated when the config is reinitialized (e.g. from a `reload`) and we don't have to restart the server to update them.
```suggestion
            App.getConfig().getList("chat.customEmoji").unwrapped();
```